src/components/VueSearchbox.jsx

```diff
@@ -86,14 +104,209 @@ const VueSearchbox = {
           fuzziness,
           searchOperators
         });
+        this.searchBase.subscribeToStateChanges(this.setStateValue, [
+          "suggestions"
+        ]);
+
+        this.searchBase.onValueChange = nextValue => {
+          this.currentValue = nextValue;
+          this.$emit("valueChange", nextValue);
+        };
       } catch (e) {
         console.error(e);
```

Comment:
We don't have `renderError` or `onError` over here ?

Reply:
it's in the subsequent PR